src/client/app_configuration_client.rs

```diff
@@ -184,7 +184,7 @@ impl AppConfigurationClient {
             .collect())
     }
 
-    pub fn get_feature(&self, feature_id: &str) -> Result<Feature> {
+    pub fn get_feature(&self, feature_id: &str) -> Result<FeatureSnapshot> {
```

Comment:
do you think we should change the function name to `get_feature_snapshot` too?

Reply:
I don't think so, but I'm not sure. Maybe we should change `FeatureSnapshot` instead. Something like `FeatureImpl` or similar

Reply:
No, I like the type name "FeatureSnapshot" :)